docker/web-and-data/Dockerfile

```diff
@@ -37,7 +37,7 @@ RUN mvn dependency:go-offline --fail-never
 COPY $PWD /cbioportal
 RUN mvn -DskipTests clean install
 
-FROM openjdk:11-jre-slim
```

Comment:
The reason we are using `openjdk-shenandoah:8` instead of `openjdk:8-jre-slim` is: we want to keep using shenandoah GC, and `openjdk:8-jre-slim` doesn't support it.